storage/mysql/log_storage.go

```diff
@@ -695,11 +696,11 @@ func (t *logTreeTX) getLeavesByRangeInternal(ctx context.Context, start, count i
 			break
 		}
 		var err error
-		leaf.QueueTimestamp, err = ptypes.TimestampProto(time.Unix(0, qTimestamp))
+		leaf.QueueTimestamp = timestamppb.New(time.Unix(0, qTimestamp))
```

Comment:
Can skip CheckValid for these as I think they're just indicators.

Reply:
I added these checks back in. I don't want to change logic here where possible.